testdata/p4_16_samples_outputs/pna-action-selector-1.p4.spec

```diff
@@ -98,7 +98,7 @@ table as {
 	actions {
 		a1
 		a2
-		set_exception
+		set_exception @defaultonly
```

Comment:
This actually changes the semantics of the program, what change caused this?

Reply:
It almost had to have been a change in the DPDK back end code that fixed this, but I haven't tried to narrow down which of those changes it was.  The frontend and midend output files have this annotation before and after this PR.

Reply:
This was an obvious bug, see https://github.com/p4lang/p4c/pull/4992/files#diff-2dded02099b0c47be6b37c34d45999cbe59f2ed1e408e6fcc7b73455dff71caaL417

Essentially the checks there were always false as it was `IR::Annotations` that was queried, and not the original node.